lib/SILGen/SILGenApply.cpp

```diff
@@ -1615,25 +1615,6 @@ static SILValue emitRawApply(SILGenFunction &SGF,
                          normalBB, errorBB);
     SGF.B.emitBlock(normalBB);
   }
-
```

Comment:
+1! +1! +1! I have wanted to remove this code for ever! Did you run a -Onone full test? That is what stopped me previously from eliminating this. Perhaps I fixed all of those uniqueness check failing cases when I caused the same bug in a different manner.

Reply:
For those listening in, this is the final remaining part of the +0 self hack left in the codebase. I feel absolved of my sins! Thank you Saint Trick!

Reply:
For those listening in, when I spoke of the full -Onone test, I meant a full test/validation-test with a debug stdlib. Normally testing/validation-testing happens with the tests being run at -Onone with a -O stdlib.

Reply:
*full test/validation-test with a debug+asserts stdlib... *sigh*.